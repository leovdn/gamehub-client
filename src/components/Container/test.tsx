import { Container } from './index'
import { renderWithTheme } from 'utils/tests/helpers'

describe('<Container />', () => {
  it('should render the heading', () => {
    const { container } = renderWithTheme(
      <Container>
        <h1>Test</h1>
      </Container>
    )

    expect(container.firstChild).toHaveStyleRule('max-width', '130rem')

    expect(container.firstChild).toMatchSnapshot()
  })
})
