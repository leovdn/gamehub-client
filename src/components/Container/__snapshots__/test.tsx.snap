// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container /> should render the heading 1`] = `
.c0 {
  max-width: 130rem;
  margin-left: auto;
  margin-right: auto;
  width: 100%;
  padding-left: calc(3.2rem / 2);
  padding-right: calc(3.2rem / 2);
}

<div
  class="c0"
>
  <h1>
    Test
  </h1>
</div>
`;
